### ISSUE TITLE
[Feature Request] Pause / Unpause speedtests

### ISSUE THREAD

**Issue by eihns** [henrywhitaker3/Speedtest-Tracker#420]
**Describe the feature**
Hi there really nice docker! Installed it on unraid and literally worked just right away... like... wow!

I have a unstable internet so i would like to disable speedtests when certain devices are online. I have everything working for that... but docker start / stop doesnt seem to be clever, because when some devices are online... i cant see the speedtest because docker is stopped, so could you give me some sort of ip:port/?(un)pause

So i can trigger start and stop of testing?

**Comment by henrywhitaker3** [comment]
Try the latest `dev` tag for this mate

**Comment by eihns** [comment]
> Try the latest `dev` tag for this mate. It's in Settings -> General -> Schedule Enabled

Hey,
thanks.

Sorry i cant find out how this helps me.

I want to start / stop testing from outside.

**Comment by henrywhitaker3** [comment]
If you uncheck that box and save, then speedtests will only run when you click the test again button. Can look at adding an API endpoint for this when I'm next working on it

**Comment by eihns** [comment]
Okay.. now i understand what you mean.

That would make it kinda hard for me.

I mean, i would need to make a own clock to start tests every 1h.

What about users can set time frames in settings (like 1h, 30min, what ever) and a simple "start and pause" function (maybe with a extra api for individual tests) 

This way you would have everything covered someone need

**Comment by henrywhitaker3** [comment]
Not really sure what you are getting at here. If you're running tests every hour manually, why not just use the Cron scheduler?

If you want to run them manually, you can make a GET request to `/api/speedtest/run` and it will queue a new test.

Will have a think about adding a "pause for x minutes" type thing though

**Comment by eihns** [comment]
Dont make it so complicated. I literally only want a way to enable and disable the checkbox for "Schedule enabled [ ]" (found under /settings)

**Comment by henrywhitaker3** [comment]
All you need to do is check/uncheck that box and click save. Instead of stopping the container, just uncheck the box and save

**Comment by eihns** [comment]
yeah, i understand that, but im asking your if you could gimme some sort of "/api/speedtest/sheduler/on" or "/api/speedtest/sheduler/off"

**Comment by henrywhitaker3** [comment]
Ah, I got you, yeah will add this in later today (sorry it's been so long)
